TsRandomizer/Screens/SaveSelectScreen.cs

```diff
@@ -318,9 +318,57 @@ static void WriteItemListSection(StreamWriter file, IEnumerable<ItemLocation> it
 		static void WriteItemList(StreamWriter file, IEnumerable<ItemLocation> itemLocations, int depth)
 		{
 			var prefix = new string('\t', depth);
+			var itemName = "";
```

Comment:
not sure why you need these varables if you only want to insert them inside the TimeSpinnerGame.Localizer.Get($"inv_{itemType}_{item}"); then you can just do that without declaring a variable

Reply:
Fixed. Apologies. Initially all of the itemNames were populated in a single line at the end until orbs threw a wrench in that. Cleaned it up.